### PR TITLE
chore(ci): add coderabbit settings

### PR DESCRIPTION
**Pull request by crodriguezvega**
## Description

Let's unleash the 🐰!

closes: #XXXX

### Commit Message / Changelog Entry

```text
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against the correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

## Summary by CodeRabbit

- **New Features**
	- Enhanced control over language settings, review workflows, and chat automation with the introduction of `.coderabbit.yml` configuration.
	- Improved path-specific instructions for code and documentation reviews.
	- Added auto-review settings for more efficient code reviews.

### CODE REVIEW

File: .coderabbit.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+language: "en"
+early_access: false
+reviews:
+    request_changes_workflow: false
+    high_level_summary: true
+    poem: false
```
Comment (chatton) [inline .coderabbit.yml:6]: :thinking: now I'm curious as to what this is :D

File: .coderabbit.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    review_status: true
+    collapse_walkthrough: true
+    path_instructions:
+        - path: "**/*.go"
```
Comment (chatton) [inline .coderabbit.yml:10]: can we exclude *.pb.go files from review?

**Comment by crodriguezvega** [comment]
> overall lgtm, we can tweak settings as appropriate later on. Wonder if these setting cover PRs opened in draft mode (I see WIP in titles as a filtering mechanism) but having it not go ape when in draft might be a good idea.

Yeah, I prefer to keep the rabbit out of the draft PRs for now. Since they are not ready for review, it might just be a bit noisy.

File: .coderabbit.yml (hunk 1 above)
Comment (chatton) [reply .coderabbit.yml:6]: and you set it to false! :scream:

**Review by chatton — APPROVED** [review]
LGTM! Thanks @crodriguezvega :muscle:

File: .coderabbit.yml (hunk 1 above)
Comment (crodriguezvega) [reply .coderabbit.yml:6]: I know, I know... It was tough decision... :)

**Review by damiannolan — APPROVED** [review]

**Pull request merged**